modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18240,6 +18240,7 @@ thousands-delimiter=,
 thousands-separator=Thousands Separator
 thread=Thread
 thread-as-question-by-default=Thread as Question by Default
+thread-cannot-be-deleted.-anonymize-instead=Thread cannot be deleted. Anonymize instead.
```

Comment:
@austinchiang What do you think about this piece of text? In particular, the "Anonymize instead" part sounds a bit weird to me, but I'm not sure.

Reply:
Maybe "The thread cannot be deleted. Anonymize it instead." or "The thread cannot be deleted. Please anonymize it."